src/main/docker/central-server-config/docker-config/application.yml

```diff
@@ -7,7 +7,7 @@ jhipster:
   security:
     authentication:
       jwt:
-        secret: my-secret-key-which-should-be-changed-in-production-and-be-base64-encoded
+        base64-secret: bXktc2VjcmV0LWtleS13aGljaC1zaG91bGQtYmUtY2hhbmdlZC1pbi1wcm9kdWN0aW9uLWFuZC1iZS1iYXNlNjQtZW5jb2RlZAo=
```

Comment:
Why do we add this here if it needs to be replaced in production anyway? For development?

Reply:
The difference is between secret and base64-secret. Since it's base64-secret it has to be something that is encoded and the previous value is not valid. I can add a comment that this should be changed in production to enhance security

Reply:
Okay, I was thinking a bit slow there. Makes sense ^^